clinica/pipelines/dwi_preprocessing_using_t1/dwi_preprocessing_using_t1_utils.py

```diff
@@ -98,42 +109,59 @@ def ants_warp_image_multi_transform(fix_image, moving_image, ants_warp_affine):
     return out_warp
 
 
-def rotate_bvecs(in_bvec, in_matrix):
-    """Rotate the input bvec file accordingly with a list of matrices.
+def rotate_b_vectors(b_vectors_filename: str, matrix_filenames: list) -> str:
+    """Rotate the B-vectors contained in the input b_vectors_filename file
```

Comment:
```suggestion
    """Rotates the B-vectors contained in the input b_vectors_filename file
```

Reply:
Actually, imperative style is usually considered more Pythonic (see [PEP-257](https://peps.python.org/pep-0257/) for more information).

Reply:
Alright ! Feels strange and unnatural, but it is indeed what they say !